docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2606,28 +2871,31 @@ When the value of the `debug` property is `true`, only information about this se
 
 **Enabling global debug:**
 
-When the value of the `debug` property is `global`, the console will display information about all ExtendedCss selectors that have matches on the current page, for all ExtendedCss rules from any of the enabled filters.
+When the value of the `debug` property is `global`, the console will display information about all extended CSS selectors that have matches on the current page, for all the rules from any of the enabled filters.
```

Comment:
If we want to substitute `ExtendedCSS` with `extended CSS`, we should do it everywhere, I believe. There are a lot of instances of `ExtendedCss` left in this article.

Reply:
there are a lot of instances of `extended CSS selectors`, exactly in such form

and it was the only case of `ExtendedCss selectors`